src/index.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 import { ApolloProvider } from 'react-apollo'
+import 'notification-polyfill'
```

Comment:
Do we need to import this in index.js or can we isolate this to the notifications module?

Reply:
I think both ways are alright, traditionally we import all polyfills at first, but if imported in the related module, it also makes sense.